README.md

```diff
@@ -79,6 +80,9 @@ To illustrate, `bin/rails console` will run the console in the docker container
 - [ExceptionHunter](https://github.com/rootstrap/exception_hunter) for exception tracking
 - [Factory Bot](https://github.com/thoughtbot/factory_bot) for testing data
 - [Faker](https://github.com/stympy/faker) for generating test data
+- [flipper](https://github.com/jnunemaker/flipper) for feature flag support
+- [flipper-active_record](https://github.com/jnunemaker/flipper) for Flipper active_record support
+- [flipper-ui](https://github.com/jnunemaker/flipper) for UI-based feature flag management
```

Comment:
I'm wondering if we should have a `/docs` directory to briefly explain how to access the visual Flipper interface or other relevant items about how / why to use Flipper. Same applies to new features.